### PR TITLE
Improves README.

### PR DESCRIPTION
**Pull request by francocipollone**
Part of https://github.com/maliput/maliput_documentation/issues/100

### Summary
This PR improves the README file:

- Providing information about installation via binaries.
- Providing information about installation via sources.
- Linking to online documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,120 @@
...
-   ```sh
-   colcon build --packages-up-to delphyne_gui --cmake-args " -DBUILD_DOCS=On"
-   ```
+ - Supports maliput_multilane and maliput_malidrive backends.
+   - A XODR or YAML file could be selected to be visualized
+ - Supports loading a Traffic Light Book.
+ - Supports loading a RuleRegistry Book.
+ - Supports loading a Rule Book.
+ - Supports loading a Phase Ring Book.
+   - Phases can be switched via the interface.
```
Comment (agalbachicar) [inline README.md:30]:
```suggestion
   - Phases can be switched via the interface when the backend implementation is the [ManualPhaseProvider](link to ManualPhaseProvider).
```

**Review by agalbachicar — APPROVED** [review]
LGMT

File: README.md (hunk 1 above)
Comment (agalbachicar) [reply README.md:30]: IRC, ManualPhaseProvider allowed to change the phase and we had a widget to do so. That is not part of maliput API but a custom backend implementation.

Comment (francocipollone) [reply README.md:30]: Sorry I haven't explained more. 

In the MaliputViewerPlugin we let the user to select the phase/phase ring they are interested in, however we are not actually "changing" the phase, we are just pairing the traffic lights and rules according to the phase's content. The outcome is the same, However, doing this that way, there is no need to "dynamic cast" to a manual phase provider in order to set the phase. We are just using the PhaseRing and Phase api to introspect the phase and update lane's rules and traffic lights accordingly. So, only maliput api is used, no base implementation is a requirement.

Comment (francocipollone) [reply README.md:30]: @agalbachicar

Comment (agalbachicar) [reply README.md:30]: Thanks!

**Review by agalbachicar — APPROVED** [review]
LGTM

**Pull request merged**